Gemfile.lock

```diff
@@ -249,3 +250,6 @@ DEPENDENCIES
   tzinfo-data
   uglifier
   web-console
+
+BUNDLED WITH
+   2.5.14
```

Comment:
Please remove this if possible, we don't want to keep this around. bundler has the tendency to update or downgrade itself without politely asking (we'd say no anyway, we want to use system bundler distributed in the containers) when installing from a lockfile that has a `BUNDLED WITH`:
https://bundler.io/blog/2022/01/23/bundler-v2-3.html

And since bundler version can change in teeny Ruby bumps (for example 3.3.0 to 3.3.1), it'd only be more headaches for us down the road or worse, we'd no longer be testing what came with our Ruby.

Reply:
Removed that part in last commit. Ran tests locally again, everything passed, so I think we good to go if there is nothing else.